tests/abstract_loader_test.py

```diff
@@ -39,7 +39,7 @@ def setUpClass(cls):
     def _search_for_bundle(self, bundle_uuid):
         # Search for the bundle uuid in the DSS and make sure it now exists and uploading was successful
         search_results = self.dss_client.post_search(es_query={'query': {'term': {'uuid': bundle_uuid}}}, replica='aws')
-        assert search_results['total_hits'] > 0
+        assert search_results['total_hits'] > 0, 'Not found'
```

Comment:
Would it be more informative to have something like `f"AWS bundle uuid not found in DSS: {bundle_uuid}"`?